### ISSUE TITLE
Add VoiceOver support

### ISSUE THREAD

**Issue by wernjie** [FlickType/FlickTypeKit#4]
It appears that there's a lack of VoiceOver support when using the FlickType SDK, whereas the official watch app supports VoiceOver just fine. This was reported by a VoiceOver user of one of my apps.

Can VoiceOver support be included in the SDK? Thanks.
